pkg/publisher/webhookPublisher.go

```diff
@@ -80,52 +80,53 @@ func (p *WebhookPublisher) svc() {
 
 func (p *WebhookPublisher) makeHttpRequest(r *publishRequest) {
 	url := p.baseUrl + "/" + strings.TrimPrefix(r.target, "/")
-	p.logger.Info("making HTTP request", zap.String("url", url))
+
+	msg := "making HTTP request"
+	level := zap.InfoLevel
+	fields := []zap.Field{zap.String("url", url), zap.String("type", "publish_request")}
+
+	// log once for this request
+	defer func() {
+		if ce := p.logger.Check(level, msg); ce != nil {
+			ce.Write(fields...)
+		}
+	}()
 
 	var buf bytes.Buffer
 	buf.WriteString(r.body)
 
 	// Create request
 	req, err := http.NewRequest("POST", url, &buf)
```

Comment:
Though it's not included in changes, would you mind to change `POST` into `http.MethodPost`?

Reply:
I have change POST int http.MethodPost in the latest commit